web/modules/custom/foreningsmentor/foreningsmentor.module

```diff
@@ -349,7 +349,7 @@ function foreningsmentor_entity_presave(
   if ($entity->getEntityTypeId() == 'user') {
     // @TODO: Change this since it blocks site-install.
     if ($entity->id() == 1 && $currentUser->id() != 1) {
-      throw new \Exception(t('Permission denied.'));
+      //throw new \Exception(t('Permission denied.'));
```

Comment:
Maybe remove the hook instead

Reply:
This shouldnt have been committed.

Reply:
I'm not sure it can be deleted. I don't understand it's purpose. It looks as to prevent users other than user 1 to edit user 1. But why do we care about that?